lms/static/scripts/postmessage_json_rpc/server/server.js

```diff
@@ -63,6 +63,12 @@ export default class Server {
     if (!this._isJSONRPCRequest(event)) {
       return;
     }
+    // Save the last reference used to the sidebar window so we can
+    // send messages back at a later time in the lms app.
+    window._sidebarWindow = {
```

Comment:
This is not my first choice with where to store and look up the frame for the sidebar. I ran into some difficulty because the rpc server is simply injected into the code via a <script> tag and executed at that time ahead of the lms app. I attempted to manipulate the import into some sort of singleton class, but could not get it to only instantiate once and remain compatible in its original capacity with the <script> tag

Ideally, I wanted to store this sidebar frame in the server instance (e.g. `this.sidebarWindow`) and have a getter that could later be used in `BasicLtiLaunchApp.js` if we could import `postmessage_json_rpc/server` so we don't have to pollute the global scope like this. But it didn't work -- I'm open to suggestions?